tests/test_scripts.py

```diff
@@ -1,22 +1,31 @@
 import os
 from entity_resolver.scripts import run
+import json
 
+# class TestArxiv:
+#
+#     def test_dummy(self):
+#         file_path = 'file.txt'
+#         run(['norm-arxiv', '', file_path])
+#         with open(file_path, 'r') as f:
+#             assert f.read() == 'arxiv'
+#         os.remove(file_path)
```

Comment:
Remove the commented codes and also note that there should be exactly 2 empty lines between import statements and the first line of code: `class TestCiteseer`